### PR TITLE
add FPC lenfy device support

### PR DESCRIPTION
**Pull request by jimzhang2**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/fpc/fu-fpc-device.c [hunk 1]
```diff
@@ -59,6 +60,14 @@
 
 #define FU_FPC_DEVICE_FLAG_RTS_DEVICE (1 << 2)
 
+/**
+ * FU_FPC_DEVICE_FLAG_LENFY_DEVICE:
+ *
+ * Device is a LENFY MOH device
```
Comment (hughsie) [inline plugins/fpc/fu-fpc-device.c:66]: can you expand out what LENFY means?

File: plugins/fpc/fu-fpc-device.c [hunk 2]
```diff
@@ -222,7 +232,10 @@ fu_fpc_device_setup_version(FuFpcDevice *self, GError **error)
 		}
 
 		data = g_malloc0(data_len);
-		if (!fu_fpc_device_fw_cmd(self, FPC_CMD_GET_STATE, data, data_len, TRUE, error))
+		if (fu_device_has_private_flag(FU_DEVICE(self), FU_FPC_DEVICE_FLAG_LENFY_DEVICE)) {
+			cmd_id = FPC_CMD_GET_STATE_LENFY;
+		}
```
Comment (hughsie) [inline plugins/fpc/fu-fpc-device.c:237]: no {} needed here

File: plugins/fpc/fu-fpc-device.c (hunk 1 above)
Comment (jimzhang2) [reply plugins/fpc/fu-fpc-device.c:66]: Lenfy is our inner project name.
I'll change it.

File: plugins/fpc/fu-fpc-device.c (hunk 2 above)
Comment (jimzhang2) [reply plugins/fpc/fu-fpc-device.c]: fixed

**Review by hughsie — APPROVED** [review]

**Pull request merged**